scene/animation/animation_player.cpp

```diff
@@ -1064,6 +1064,10 @@ void AnimationPlayer::_animation_process_data(PlaybackData &cd, double p_delta,
 			end_notify = prev_pos > 0; // Notify only if not already at the beginning.
 		}
 	}
+
+	if (looped_flag != Animation::LOOPED_FLAG_NONE) {
+		emit_signal(SceneStringNames::get_singleton()->animation_looped, playback.assigned);
+	}
```

Comment:
I faintly remember that the reason we did not implement this in #70278 before was that depending on the length of the animation, this might not fire correctly.

And I remember that these flags were only "populated" for the purpose of calculating looped time, and that the flags were not always fired correctly during the loop. Implementing this would require more strict testing, so if you want to focus on audio for now, this should be removed and a separate PR should be used.

Reply:
I'll probably just remove it for now, then.